src/drizzle/schema.ts

```diff
@@ -1066,3 +1066,13 @@ export const itemTagsTable = pgTable(
     unique('UQ_item_tag').on(table.tagId, table.itemId),
   ],
 );
+
+export const maintenanceTable = pgTable(
+  'maintenance',
+  {
+    slug: varchar({ length: 100 }).notNull(),
+    startAt: timestamp('start_at', { withTimezone: true, mode: 'string' }).notNull(),
+    endAt: timestamp('end_at', { withTimezone: true, mode: 'string' }).notNull(),
+  },
+  (table) => [unique('maintenance_slug_key').on(table.slug)],
```

Comment:
Any specific reason why you added the constraint as a table constraint instead of directly on the column definition ? 
I think the following is valid:
```ts
slug: varchar({ length: 100 }).notNull().unique(),
```

Reply:
I see, I didn't know. I did the same as for all the others tables. I changed it.